locale/translations/en_US.json

```diff
@@ -81,6 +81,7 @@
     "page.edit_user.title": "Edit User: %s",
     "page.feeds.title": "Feeds",
     "page.feeds.last_check": "Last check:",
+    "page.feeds.unread": "Unread:",
```

Comment:
I'm not sure how to deal with localization, is it okay for me to only add English version, or how does it work?

Reply:
I've seen in other PRs you ask people to update all translations so I did that (and it's not like it's the first time the word "unread" is mentioned in miniflux anyway 😄)